test/worker_test.rb

```diff
@@ -21,13 +21,11 @@ def messages
 
   before do
     @worker = Resque::Worker.new(:jobs)
+    Resque::Worker.any_instance.stubs(:register_signal_handlers)
+    Resque::Worker.any_instance.stubs(:unregister_signal_handlers)
     Resque::Job.create(:jobs, SomeJob, 20, '/tmp')
   end
 
-  after do
-    @worker.kill_background_threads
```

Comment:
this is unnecessary now, since https://github.com/resque/resque/pull/1478 takes care of it